### DISCUSSION TITLE
save information card

### DISCUSSION THREAD

**Discussion by peppect84** [MatusKysel/EMVemulator#19] [General]
hello, after completing the reading, if the apk is closed it does not save the data and it is necessary to read the card again, with the good probability that it will soon become unusable. is it not possible to implement the apk and have it save the read information on the device?

**Comment by MatusKysel** [comment]
No, app stores data that was collected

**Comment by peppect84** [comment]
After I closed it, the app went back to reading mode.

**Reply by MatusKysel** [reply]
yes but you can use is for emulating card without opening it. App is working in the background. Also keep in mind this is just a proof of concept app, so it doesn't have perfect UX

**Comment by peppect84** [comment]
if I try to read another card, is the one memorized lost?  I tried to pass it on a pos in reading mode but nothing happens, while when I tried to complete reading it passed as if it were my card.

**Reply by MatusKysel** [reply]
yes, it's lost. App keeps only data from last scan. You should not have it in reading mode - check the readme
```4, Now you can put this app to background or just close it and whenever you want to reply collected data just turn on your screen and app will start to communicate with reader```

**Comment by peppect84** [comment]
unfortunately after closing the app it went back to reading mode, so it had to stay with the card data saved like the one in the picture?
https://github.com/MatusKysel/EMVemulator/raw/master/pic/Completed.png
can i do something to fix it?
